jobs/README.rst

```diff
@@ -1,35 +1,45 @@
-This folder contains several worker jobs.
+This folder contains several scripts to aggregate build information from different sources and keeping it up to date.
 
 .. notes::
 
-    The ``user:pass`` is the Basic auth on Kinto.
+    The ``user:pass`` in the command-line examples is the Basic auth for Kinto.
 
 
-Pulse listener
-==============
+Scrape archives
+===============
 
-Listen to Pulse build and publishes records on a ``builds`` collection.
+Scrape releases on https://archives.mozilla.org and publishes records on a ``archives`` collection.
 
-Obtain Pulse user and password at https://pulseguardian.mozilla.org
 
 .. code-block:: bash
 
-    PULSEGUARDIAN_USER="my-user" PULSEGUARDIAN_PASSWORD="XXX" python2 listen_pulse.py --auth user:pass --debug
+    python3 scrape_archives.py --server http://localhost:8888/v1 --auth user:pass --debug
 
 
-Scrape archives
-===============
+System-Addons updates
+=====================
 
-Scrape releases on https://archives.mozilla.org and publishes records on a ``archives`` collection.
+Fetch information about available system addons updates for every Firefox release.
```

Comment:
I only just realized that these are not system addons (per the field name), but rather system addons updates. Maybe something like `Add information to each `archive` record detailing the system addons for that release. Some are built in, but others are updated from AUS.` Maybe also `Each addon has its ID, a builtin version (if any), and an update available from AUS.`